src/components/CategoriesContainer.js

```diff
@@ -38,52 +41,33 @@ export const CategoriesContainer = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  const selectCategory = (name) => {
-    dispatch(setIsSelected(name));
-    // setApprovedCount(approvedCount + 1);
-  };
-
   return loading ? (
```

Comment:
Nested logical operators is a bad practice =)
it can be used in simple cases but not more than 2 levels.
For react components better to have simple returns:
```
if ( loading ) return <h2>Loading...</h2>
if ( error ) return <h2>{error}</h2>

return ( 
  <div className='wrapper'>
  ...
  </div>
);